### PR TITLE
Refactoring of code

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

File: src/pybrake/metrics.py [hunk 1]
```diff
@@ -5,14 +5,12 @@
...
 import time as pytime
 
+from . import constant
 from .utils import logger
 
 
 threadLocal = threading.local()
 
-FLUSH_PERIOD = 15
-_METRIC_KEY = "_ab_metric"
```
Comment (kyrylo) [inline src/pybrake/metrics.py:14]: I don't think it's a great idea to move all the constants into 1 file. It looks like we don't use this constant elsewhere, so why not keep it in this file? Same for other constants that are not re-used. Let's keep them in those files where they are used unless they're shared.

File: src/pybrake/metrics.py [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 import time as pytime
 
-from . import constant
 from .utils import logger
 
 
 threadLocal = threading.local()
 
+FLUSH_PERIOD = 15
+_METRIC_KEY = "_ab_metric"
```
Comment (kyrylo) [inline src/pybrake/metrics.py]: I don't understand what's going on. A couple commits ago we moved this to `constants` and now we are moving it back?

**Review by kyrylo — COMMENTED** [review]
I don't understand the changes here. Why did we reintroduce FailedStats and then renamed them?

File: src/pybrake/metrics.py (hunk 1 above)
Comment (smurf-U) [reply src/pybrake/metrics.py:14]: Moved back all private variable which use in single file. Like:- _METRIC_KEY, _ERR_IP_RATE_LIMITED, _AB_URL_FORMAT.

**Comment by smurf-U** [comment]
PTAL

**Review by kyrylo — APPROVED** [review]

**Pull request merged**